backend/game/irepository/irepository_player.py

```diff
@@ -0,0 +1,34 @@
+import datetime
+
+from game.models import User
+from game.player.player import Player
+
+
+class IRepositoryPlayer:
+    def __init__(self):
+        self.user = None
+
+    def save_player_db(self, player: Player):
+        self.user = User(monster_name=player.monster_name,
+                         username=player.username,
+                         password=player.password,
```

Comment:
Don't think passwords should be save plaintext see review on #80

Reply:
rectified by using password=md5.encrypt(player.password),

Reply:
password reference removed

Reply:
will have to deal with passwords eventually... but agree with removing for the moment.